model_cards/huawei-noah/DynaBERT_MNLI/README.md

```diff
@@ -0,0 +1,24 @@
+# DynaBERT: Dynamic BERT with Adaptive Width and Depth
+
+* DynaBERT can flexibly adjust the size and latency by selecting adaptive width and depth, and 
+the subnetworks of it have competitive performances as other similar-sized compressed models.
+The training process of DynaBERT includes first training a width-adaptive BERT and then 
+allowing both adaptive width and depth using knowledge distillation. 
+The overview of DynaBERT learning is shown below. 
+
+<img src="dynabert_overview.png" width="800" height="320"/>
```

Comment:
Generally speaking, I don't think it's a good idea to put images inside the model card directory. When cloning, these static files will have to be downloaded as well by all cloners which can cause an extra burden. I suggest putting these files on your own github repository or static server ([example](https://raw.githubusercontent.com/huggingface/transformers/master/examples/bert-loses-patience/README.md))

Reply:
Thank you for your suggestion. I'll make changes accordingly.